src/slate-react/MaterialSlate.jsx

```diff
@@ -27,7 +27,10 @@ export default function MaterialSlate({
   const classes = useStyles()
   return (
     <Slate value={value} editor={editor} onChange={value => onChange(value)}>
-      <Box className={classes.root} borderRadius="borderRadius">
+      <Box
+        className={`${className} ${classes.root}`}
```

Comment:
It should be the opposite. if not, classes.root will overwrite whatever the className sets...

Reply:
@merlos Yes, it only has border, if they want they can override the class. Don't we allow them to do it?

Reply:
the first class (className) will be overwritten by the second class (classes.root) if I want to have a blue border, I won't be able. 
Please change the order

Reply:
yes, your correct. I thought the other way .